core/src/main/scala/org/ensime/indexer/domain.scala

```diff
@@ -22,6 +22,8 @@ object Access {
     else Default
 }
 
+final case class FullyQualifiedReference(fqn: FullyQualifiedName, line: Option[Int])
```

Comment:
ok, this is probably a good thing. Is it always obvious which source file it refers to?